.github/workflows/test.yml

```diff
@@ -27,12 +27,23 @@ jobs:
         sudo apt-get install -y cmake g++ libhdf5-openmpi-dev librange-v3-dev ninja-build nlohmann-json3-dev
         sudo apt-get install -y libarrow-dev libparquet-dev
 
-    - name: Configure
+    - name: Configure and build
       run: |
         cmake -B build -S . -GNinja \
+          -DCMAKE_INSTALL_PREFIX=$PWD/install \
           -DCMAKE_CXX_COMPILER=$(which mpicxx)
         cmake --build build
         ctest --test-dir build --output-on-failure
+        cmake --install build
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+
+    - name: Small integration test
+      run: |
+        export PATH=$PATH:$PWD/install/bin
+        python -mpip install -r tests/test_requirements.txt
+        python tests/test_integration.py
```

Comment:
It might pay off to immediately start using `pytest`.

Reply:
Well, it's already set up for that :)